site/content/entry/citra-progress-report-2017-october-november.md

```diff
@@ -0,0 +1,99 @@
++++
+date = "2018-01-10T13:11:00-04:00"
+title = "Citra Progress Report - 2017 October and November"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 0
++++
+
+Past the red leaves and into the icy tundra in the midst of winter comes the present.
+This time we'll be covering not one, but two months of progress! With a great many
+changes all around, this article is going to be packed to the brim with all the
+new goodies that have come to Citra those crimson months.
+
+But, enough faffing about! Let's get right into it:
+
+## [citra-qt : Adding fullscreen mode](https://github.com/citra-emu/citra/pull/3001) by [Styleoshin](https://github.com/Styleoshin)
+
+One of the most requested features for Citra, second only to controller support,
+has been enlarging the window to cover the entirety of the screen. After almost
+a year of requests on both the Citra Discourse and Discord servers, [Styleoshin](https://github.com/Styleoshin)
+has finally delivered!
+
+{{< figure src="/images/entry/citra-progress-report-2017-october-november/fullscreen.png" 
+    title="No more pesky window décor!" >}}
+
+Simply go into `View → Fullscreen`, or just strike the Alt + Return keycombo, and
```

Comment:
"just strike the _Alt + Return_ keycombo"

This is F11 in Windows.

Reply:
both works iirc (i can double check later)